projects/valtimo/components/src/lib/components/form-io/documenten-api-uploader/documenten-api-uploader.component.scss

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright 2015-2020 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.progress-bar-container {
+  padding: 15px 20px;
+}
+
+::ng-deep .formio-component-valtimo-file {
+  .formio-button-add-another,
+  .btn-secondary {
+    display: none;
+  }
+
+  td {
+    padding: 0 !important;
+    border: none;
+  }
+
+  .table {
+    border-collapse: collapse;
+    border: none;
+    margin: 10px 0;
+  }
+}
+
+.dropzone-container {
+  margin-bottom: 12px;
+}
+
+.file-preview {
+  margin-top: -1px;
+  display: flex;
+  border: 1px solid #dee2e6;
+
```

Comment:
`#dee2e6`
maybe create a variable for this color, to reuse. It is used in several places on the **border** property

![Screenshot 2022-09-27 at 09 47 15](https://user-images.githubusercontent.com/110469681/192480474-8fe77abb-0a97-44bd-a1be-8ae51ea046d8.png)

Reply:
This uses the old design system (Bootstrap/Beagle), we don't use design variables for that